BoxContentSDK/BoxContentSDK/Helper/BOXRepresentationsHelper.h

```diff
@@ -0,0 +1,22 @@
+//
+//  BOXRepresentationsHelper.h
+//  BoxContentSDK
+//
+//  Created by Prithvi Jutur on 8/24/18.
+//  Copyright © 2018 Box. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface BOXRepresentationsHelper : NSObject
+/**
+ Setting a representation or list option will include availability of file with request representation information
+ */
+@property (nonatomic, readwrite, strong) NSMutableOrderedSet *representationsRequested;
```

Comment:
Seems like this could be `readonly`, since it's mutable. It strikes me as an anti-pattern exposing it as mutable, too.

Reply:
Of what is this a set?

Reply:
Again, None of this is new, just moved around.